cob_hardware_config/cob4-3/config/emergency_stop_monitor.yaml

```diff
@@ -0,0 +1,6 @@
+sound_enabled: False
+led_enabled: True
+diagnostics_based: True
+motion_based: True
+led_components: [light_base]
+color_ok: cyan
```

Comment:
does this result in warnings on the terminal?
see a default config for [cob4-4](https://github.com/ipa320/cob_robots/pull/325/files#diff-57b51d6fe80d56448bd650c9d9325367)
Do they have a light at all?